ngraph/python/src/ngraph/opset8/ops.py

```diff
@@ -99,3 +99,111 @@ def deformable_convolution(
             "bilinear_interpolation_pad": bilinear_interpolation_pad
         },
     )
+
+
+@nameable_op
+def multiclass_nms(
+    boxes: NodeInput,
+    scores: NodeInput,
+    sort_result_type: str = "none",
+    sort_result_across_batch: bool = False,
+    output_type: str = "i64",
+    iou_threshold: float = 0.0,
+    score_threshold: float = 0.0,
+    nms_top_k: int = -1,
+    keep_top_k: int = -1,
+    background_class: int = -1,
+    nms_eta: float = 1.0,
+    normalized: bool = True
+) -> Node:
+    """Return a node which performs MulticlassNms.
+
+    @param boxes: Tensor with box coordinates.
+    @param scores: Tensor with box scores.
+    @param sort_result_type: Specifies order of output elements
```

Comment:
Good thing to add: what values can be set as `sort_result_type`?

Reply:
Added.